include/cuco/detail/static_map.inl

```diff
@@ -443,7 +443,7 @@ __device__
                 "insert_and_find is not supported for unpackable data on pre-Volta GPUs.");
 #endif
 
-  auto current_slot{initial_slot(insert_pair.first, hash)};
+  auto current_slot{this->initial_slot(insert_pair.first, hash)};
```

Comment:
I'm trying to understand the logic here since we have many places missing `this->` but how can others be built properly?

Reply:
I would be genuinely surprised if this is the only place where we forgot to use `this->`. So yeah, not sure why clang is complaining here but not in the other cases.

Reply:
My suspicion is that this is the only code I am calling in my application and exercising.  I need to go dig in and get you a real answer though, not a guess.

Reply:
> this is the only code I am calling in my application and exercising. 

Make sense.


@sleeepyjack we can merge the current PR as it is and fix other uncovered issues in your clang CI PR. What do you think?

Reply:
I'm fine with that, especially if it unblocks @AustinSchuh 's work for now.